trace_context_ext/README.md

```diff
@@ -0,0 +1,12 @@
+# Trace-Context-Ext Header
```

Comment:
this looks good in general.. nit I might title this Trace-Context-Ext *Field* as header is transport specific